test/tests/mutations/update.test.js

```diff
@@ -140,7 +140,7 @@ describe('graphql - update mutations', () => {
       mutation ($input: AdminService_Books_U!) {
         AdminService {
           Books {
-            update(input: $input) {
+            update(filter: {}, input: $input) {
```

Comment:
But isn't it required to include the keys in the filter?

Reply:
No, filters are basically where conditions with limited operators. Keys don't necessarily need to be specified.

Reply:
According to the change log : 
 Empty filter lists resolve to `false` and empty filter objects resolve to `true`

Reply:
So in this case it would update all entries in the database, do we really want to allow that? Or are there checks in our runtime?

Reply:
Yes it would update all entries and there are no checks in the runtime. We discussed this with @ralfhandl and determined that there will always be ways to cause a filter to affect most/all entries, since it is essentially a `where` condition. Making the `filter` argument mandatory is a step against accidentally affecting all entries by forgetting the argument.